### ISSUE TITLE
Milestone 6

### ISSUE THREAD

**Issue by marpierc** [airavata-courses/TeamAlpha#45]
Jenkins worked, and I was able to create a new account, but Karst job submission doesn't seem to work.

**Comment by patel99** [comment]
Hi Marlon,

Did you got any error msg on console or logs? Just to confirm, have you put your public key of VM on karst? We have not automated that process yet.

**Comment by marpierc** [comment]
No, I didn’t upload my key.  But you can upload your key and I can run the job as you.

**Comment by patel99** [comment]
Our gateway takes private key from the path mentioned in this file - https://github.com/airavata-courses/TeamAlpha/blob/master/Airavata_Remote_Job_Runner/src/main/resources/user_input.properties,

I have already added my public key on karst and bigred 2, but to make it work you would need my private key. What I would suggest is, create a pair of private-public pair on vm and just copy this new public key to karst.

**Comment by patel99** [comment]
Hi Marlon,

I just checked 'http://js-168-234.jetstream-cloud.org:8080/jenkins/'. Is this the jenkins server that you are using?

You will need to create 4 jobs for milestone 6.
1. Same job as Milestone-5, you would just need to change branch name.
2. Job to deploy user registration, this would be completely new job
3. Job to monitor the submitted jobs 
4. Job to save metadata for jobs on DB

If you are using the same jenkins server as I mentioned above, then I could help you with configuring all the jobs as I do have the access to the jenkins. Let me know what you would like us to do

**Comment by marpierc** [comment]
I can do a google hangout in a few minutes unless you are already on your way to the CIB.  Are the instructions below in GitHub?

**Comment by marpierc** [comment]
That is the correct URL.

**Comment by patel99** [comment]
Yes instructions are at - https://github.com/airavata-courses/TeamAlpha/wiki/6.-Project-Milestone-6:-Details-and-Instructions

Lets do a hangout whenever you are free

**Comment by marpierc** [comment]
I’m available now: http://s.apache.org/airavata_hangout
